components/schemas/scoped-variables/URLSource.yml

```diff
@@ -15,14 +15,16 @@ properties:
     required:
       - url
       - headers
+      - auth_token_url
     properties:
       url:
         type: string
         description: The URL to call to produce the value.
       headers:
         type: object
-        description: Additional information about a container
+        description: Additional information about a URL source
         additionalProperties: {}
       auth_token_url:
         nullable: true
         type: string
+        description: The response is from auth_token_url will be sent as header authorization which allows third party integration
```

Comment:
This comment needs some work. The grammar is strange, and it isn't really specifying what the field is.

"An optional URL that can be provided to authenticate with a third party secret service. Cycle will make a request to this URL before fetching  the secret URL, and use the response as the value of an Authorization header when requesting the secret."

Reply:
<img width="1905" alt="image" src="https://github.com/cycleplatform/api-spec/assets/112718756/e49c151c-b4d3-4ddb-a991-dca431971deb">

Reply:
saying nullable in the description is strange. just drop it from here

Reply:
Got it.  You originally said optional, which wasn't true